src/pages/SamplePage/SamplePage.tsx

```diff
@@ -247,6 +247,14 @@ const SamplePage: React.FC = () => {
             </Banner>
           )}
 
+        {!schema.data && !schema.isFetching && (
+          <Banner color="danger">
+            <IonLabel>
+              <b>Error</b> Missing schema information
```

Comment:
I think some people will read "Missing" as an adjective as opposed to a verb, and then wonder, "What about the missing schema information?" With that in mind, I have this proposal, which I also consider to be actionable:

```suggestion
              <b>Error</b> Schema not found. Please ensure you are connected to the internet and try again.
```

This is not a deal-breaker, from my perspective, though. Also, I don't know how much screen real estate is available to the `<Banner ...>` element. The wording I'm suggesting is several times longer than the original.

Reply:
> I don't know how much screen real estate is available to the <Banner ...> element. 

It will grow to fit whatever message you give it. In this case there wouldn't be anything else on the page so there's no harm in being more verbose here.